gensim/test/test_phrases.py

```diff
@@ -378,9 +379,11 @@ def test_save_load_custom_scorer(self):
     def test_save_load(self):
         """Test saving and loading a FrozenPhrases object."""
         with temporary_file("test.pkl") as fpath:
-            bigram = FrozenPhrases(Phrases(self.sentences, min_count=1, threshold=1))
+            bigram = FrozenPhrases(Phrases(self.sentences, min_count=1, threshold=1,
```

Comment:
[Hanging indent](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#code-style) please.

Reply:
code style fix updated in the commit